tokio/src/macros/cfg.rs

```diff
@@ -131,6 +131,36 @@ macro_rules! cfg_io_driver {
     }
 }
 
+macro_rules! cfg_io_driver_or_uring {
```

Comment:
Should you not change `cfg_io_driver!` instead? It seems like you're adding a new case where the IO driver is available rather than having logic that applies when the IO driver is not available but we have io_uring.